src/App.vue

```diff
@@ -540,6 +428,59 @@ export default class App extends Mixins(
     }
   }
 
+  /** Check authorizations. */
+  async checkAuth (): Promise<void> {
+    try {
+      // load account information
+      await this.loadAccountInformation().catch(error => {
```

Comment:
I suggest you keep this separate (decouple) from checking auth. Example:

https://github.com/bcgov/business-edit-ui/blob/c3bd1aa35bc8f39a5f87a8f07d97585b372298a1/src/App.vue#L426